pkg/golinters/nolintlint/internal/nolintlint.go

```diff
@@ -128,6 +22,10 @@ const (
 	NeedsAll = NeedsMachineOnly | NeedsSpecific | NeedsExplanation
 )
 
+type Needs uint
+
+const commentMark = "//"
```

Comment:
We can use this on L137 and L205 as well (but can't comment there since it's not changed).

Reply:
The constant is now use on L137 and L205